docs/api-docs/tax/tax-properties.md

```diff
@@ -1,14 +1,19 @@
 # Tax Properties for Products
 
-The Tax Properties API allows merchants to attach multiple tax codes to a given product. These tax codes allow merchants to use more than one tax provider; for example, a merchant might use different tax providers in different countries. They can also assist merchants who are transitioning between tax providers. 
+The Tax Properties API allows merchants to vary the tax info they send to tax providers. In some jurisdictions, tax rates for products vary by product composition. For example, correct taxation of alcohol requires info about alcohol percentage. Using tax properties, providers can return more accurate tax calculations. The examples in this guide expand on this use case. 
```

Comment:
```suggestion
The Tax Properties API allows merchants to vary the tax information they send to tax providers for tax calculation. In some jurisdictions, tax rates for products vary by product composition. For example, accurate taxation of alcohol may require information about alcohol percentage. Using tax properties, providers can return more accurate tax quotes. The examples in this guide expand on this use case. 
```

🍹 Avoiding `info` abbreviated form of `information`.
🍹 Added `for tax calculation` since that first sentence seems like a great introduction sentence.
🍹 Loosened "correct" to "accurate" => correct is more black and white, whereas accuracy is more of a grey area (and speaks to tax capabilities).
🍹 Loosened the example a little since not all jurisdictions will require this (they _may_ require this).
🍹 I think "tax quote" may be a better description for what the provider returns here - that is, calculation is more the service they are providing which results in the tax quote being returned.